### PR TITLE
fix: invert longitude values

### PR DESCRIPTION
**Pull request by HelenDias**
Inverte valores de longitude no exercício

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,3 +85,6 @@ After finishing the test, send an email to vagas-dev@juntossomosmais.com.br, wit
 ## Questions
 
 If you have some questions or suggestions, send your questions directly to us (vagas-dev@juntossomosmais.com.br) or opening a issue.
+
+Please, check if your question has already been answered in [a issue](https://github.com/juntossomosmais/frontend-challenge/issues) :relaxed:
```
Comment (lupy100) [inline README.md]: Maybe showing all is better

```suggestion
Please, check if your question has already been answered in [a issue](https://github.com/juntossomosmais/frontend-challenge/issues?q=) :relaxed:
```

**Review by lupy100 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (HelenDias) [reply README.md]: Done: https://github.com/juntossomosmais/frontend-challenge/pull/4/commits/c89320c649261c0e3d3d8c5b576384ab16b6c6c9

File: README.md [hunk 2]
```diff
@@ -47,24 +47,24 @@ To optimize our service, we divided our consultants into three regions. Consider
...
-minlon: -2.196998
+minlon: -15.411580
 minlat -46.361899
-maxlon: -15.411580
+maxlon: -2.196998
 maxlat: -34.276938
 ```
 ```
-minlon: -19.766959
+minlon: -23.966413
```
Comment (hick97) [inline README.md]:
```suggestion
minlon: -20.000000
```

**Review by felipefialho — APPROVED** [review]
🚀

File: README.md (hunk 2 above)
Comment (HelenDias) [reply README.md]: Done: https://github.com/juntossomosmais/frontend-challenge/pull/4/commits/beb9cc3e3e6d6e50ddcdad60b4967babc2b54191

**Pull request merged**